.eslintrc

```diff
@@ -9,7 +9,7 @@ env:
 
 rules:
   indent: [2, 2] # 2 spaces indentation
-  max-len: [2, 80, 4]
+  max-len: [2, 100, 4]
```

Comment:
![screen shot 2015-08-11 at 10 03 56](https://cloud.githubusercontent.com/assets/157534/9193004/58e7daee-4010-11e5-8730-a6ffa031670d.png)
Not sure why you needed to change this...